--- ./main/main.cc	2021-02-26 11:14:30.731580071 +0100
+++ ./linting/main.cc	2021-02-26 14:15:54.583147905 +0100
@@ -1,13 +1,12 @@
-#include "nlohmann/json.hpp"
 #include "lib/service.h"
+#include "nlohmann/json.hpp"
 #include <iostream>
 
-
-int main(int argc, char** argv) {
+int main(int argc, char **argv) {
   nlohmann::json obj = {
       {"bazel", "https://bazel.build"},
-      {"cmake", "https://cmake.org/"},    
+      {"cmake", "https://cmake.org/"},
   };
   std::cout << obj.dump(4) << std::endl;
-  std::cout<< greet("Maroun") << std::endl;
+  std::cout << greet("Maroun") << std::endl;
 }
